sirepo_bluesky/tests/conftest.py

```diff
@@ -55,36 +57,42 @@ def RE_no_plot(db):
     return RE
 
 
-@pytest.fixture(scope='function')
-def srw_youngs_double_slit_simulation():
+@pytest.fixture(scope="function")
+def make_dirs():
```

Comment:
The pytest fixture `tmpdir` could be used to avoid interactions with an existing `/tmp/sirepo-bluesky-data` and to clean up after tests.

Reply:
But this is not relevant to the PR at hand.

Reply:
Yes, we can address it in a separate PR. This will require updating the `create_classes` factory function to accept the directory kwarg for simulation data files, as the watchpoint class is created dynamically. We can discuss if we want to create that class manually.